### ISSUE TITLE
"admin" account login doesnt work

### ISSUE THREAD

**Issue by purplemana** [SenseNet/sensenet#423]
Hi,

tldr; Not able to login using `BUILTIN\admin` in sensenet website mounted on IIS. The login works in the source application running in the same machine on IIS Express. A newly created user login works in both applications.

**The setup**
I created a new sensenet project per the documentation. I am using a SQL-Express db installed on the same machine. The site works perfectly fine when I run it from Visual Studio using IIS Epxress. I am able to login using the default admin credentials and can play around as usual. The application runs on `http://localhost:44390` and I have also changed `Default_Site` locations so that I see the login page directly on `http://localhost:44390`.

Next I published the web application from Visual Studio to a folder inside `inetpub` and hosted the website in IIS on port `80`. Then I added `localhost` to `Root/Sites/Default_Sites` locations list.

So now the web application can be browsed on `http://localhost` (IIS) as well as `http://localhost:44390` (IIS Express). Both the applications point to the same SQL-Express DB. The auth settings in IIS are set properly to allow anonymous and everything else is disabled.

**And the issue**
Now comes the weird part... on `http://localhost` the default `admin` user is not able to login at all (the login page responds with `Wrong username or password.` message). I tried typing in `BUILTIN\admin` as well but it didn't work either. The login works properly on IIS Express though.

Then using the IIS Express site, I created a new user. And the _weirder_ part is that this new login works on both IIS site as well as IIS Express.

So why is the BUILTIN/admin user not able to login through the IIS application? Any help is appreciated.

**Comment by tusmester** [comment]
Hi @purplemana, sorry for the late reply, thank you for your patience. 

When you want to publish/deploy a sensenet site, there are several things to take care of:

- **database**: this has to be unique for each installation, you should not point your local environment and a central dev/test environment to the same database for example.
- **web folder**: if you want a load balanced (NLB) environment (multiple web instances connecting to a single db), it requires some configuration, for example setting up [messaging](http://wiki.sensenet.com/Communication_between_appdomains) (currently requires MSMQ, but [RabbitMq](https://github.com/SenseNet/sn-messaging-rabbitmq) will be available soon), because web app domains need to be able to communicate with each other.
- **index**: inside the _web\App_Data_ folder there is the index. That has to be there in every web folder, but it can not be published from Visual Studio.

See this thread for more info:

https://stackoverflow.com/questions/51543037/localindex-files-are-not-copied-during-deployment-process

In your case this may be related to an incorrect/corrupt index (the query during authentication does not find the admin user in the index). You may try to re-create the index using [SnAdmin](https://community.sensenet.com/docs/snadmin-tools/).

**Comment by purplemana** [comment]
Thanks @tusmester, I will try purging the index or setting up a new website under IIS using the instructions. I will close this question soon.

**Comment by tusmester** [comment]
We are closing this issue for now. If you have more or new info, please reopen this, or create a new entry here or in stackoverflow.
